test/fluree/http_api/system_test.clj

```diff
@@ -138,12 +144,13 @@
           headers     {"Content-Type" "application/edn"
                        "Accept"       "application/edn"}
           res         (post :transact {:body req :headers headers})]
-      (is (= 200 (:status res)))
+      (is (= 200 (:status res))
+          (str "Response was: " (pr-str res)))
       (is (= {:address address, :alias ledger-name, :t -2}
              (-> res :body edn/read-string))))))
 
 (deftest ^:integration query-test
-  (testing "can query a basic entity"
+  (testing "can query a basic entity w/ EDN"
```

Comment:
I'm not suggesting we remove this at this point, but I don't see why we'd ever encourage anyone to use EDN using the http-api-server. At minimum per my comment below we shouldn't be doing anything where we are mixing the concepts of JSON and keywords/EDN here, I think any such mixing should be an exception, not a feature.